### ISSUE TITLE
ask for updating Cis-bp

### ISSUE THREAD

**Issue by wangmhan** [vanheeringen-lab/gimmemotifs#127]
**Is your feature request related to a problem? Please describe.**
Now Cis-bp has changed to version 2.0.0, while the version in GimmeMotifs is still 1.0.2

**Describe the solution you'd like**
Is it possible to use the latest version when doing motif discovery?

**Describe alternatives you've considered**
That would be a big help, as version 2.0.0 is made in 2019 with improvement.

thanks & best regards

**Comment by simonvh** [comment]
@wangmhan good point. For the CIS-BP database itself, the version is 2.00. However, the `gimme.vertebrate.v5.0` is indeed based on the older CIS-BP. I will add that to the todo!
